docs/source/gbd2017_models/risks/low_birthweight_short_gestation/index.rst

```diff
@@ -4,42 +4,40 @@
+
+Exposures and relative risks for the GBD Low birth weight and short gestation risk factors are divided
+into joint 500-gram birth weight and 2-week gestational age combinations. The lowest risk overall 500-
+gram/2-week bin is the overall TMREL. The univariate TMRELs vary with GA and BW. The lowest risk GA
+varies by BW category and the lowest risk BWs vary with GA category. The latter are used to quantify
+univariate attributable risk. Under this framework, all attributable burden under the joint TMREL is
+referred to jointly as burden of LBWSG. All attributable burden to BWs under the TMREL for each GA
+category are, on aggregate, “low birth weight” and all attributable burden to GAs under the TMREL for
+each BW category are, on aggregate, “short gestation.” Each combination of 500-grams and 2-wks is
+associated with a relative risk for mortality by neonatal period (early and late neonatal) and by the
+causes, and relative to the joint TMREL.
 
-The meaning of the “low birth weight” and “short gestation” in GBD have subtle
-definitional differences compared to other usages of “low birth weight” and
-“short gestation” in literature. The term “low birth weight” has historically
-been used to refer to birth weight (BW) less than 2500 grams. However, because
-the goal of the GBD risk factors analysis is to quantify the entirety of
-attributable burden due to each risk factor, the GBD definition of “low birth
-weight” therefore refers to all birth weight below the Theoretical Minimum Risk
-Exposure Level (TMREL) for birth weight. Likewise, new-borns have been typically
-been classified into gestational age (GA) categories of “extremely preterm” (<28
-weeks of gestation), “very preterm” (28-<32 weeks of gestation), and “moderate
-to late preterm” (32-<37 weeks of gestation). “Short gestation” in GBD refers to
-all gestational ages below the gestational age TMREL.
-
-Exposures and relative risks for the GBD Low birth weight and short gestation
-risk factors are divided into joint 500-gram birth weight and 2-week gestational
-age combinations. The lowest risk overall 500- gram/2-week bin is the overall
-TMREL. The univariate TMRELs vary with GA and BW. The lowest risk GA varies by
-BW category and the lowest risk BWs vary with GA category. The latter are used
-to quantify univariate attributable risk. Under this framework, all attributable
-burden under the joint TMREL is referred to jointly as burden of LBWSG. All
-attributable burden to BWs under the TMREL for each GA category are, on
-aggregate, “low birth weight” and all attributable burden to GAs under the TMREL
-for each BW category are, on aggregate, “short gestation.” Each combination of
-500-grams and 2-wks is associated with a relative risk for mortality by neonatal
-period (early and late neonatal) and by the causes listed in Table 2 and
-described below, and relative to the joint TMREL.
+.. image:: Doc - Apr 12 2019 - 9-04 AM.pdf
```

Comment:
This image isn't rendering. I think that there can't be any spaces in the title of the file (it needs to be Apr_12_2019, not Apr 12 2019). Also, I'm not sure if .pdfs work? I think that .svg is the desired format, but .png will also work.

Reply:
Please rename this image with a more descriptive filename (without spaces), e.g. `lbwsg_categories.pdf`. I don't know whether Sphinx/ReST will render the pdf or not, but let's try it. If it doesn't work, maybe we can figure out how to convert it to a different format.

Reply:
Ive made it into an svg and rendered!

I've made a new branch called lbwsg_risk_again that incorporates all your comments. Thank you for the thoughtful feedback :)